packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -43,25 +43,47 @@ function composeObjectExpressionPropertyIntoString(
 
 export function getLoadArgument(
   node: TSESTree.MemberExpression
-): string | undefined {
+): string[] | undefined {
   node = findTopLevelExpression(node);
 
   if (
     isLoadFunction(node) &&
     node.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression
   ) {
-    if (node.parent.arguments.length === 0) {
-      return undefined;
-    }
-    if (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal) {
-      return node.parent.arguments[0].value as string;
-    } else if (
-      node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.ObjectExpression
+    if (
+      node.parent.arguments.length === 0 ||
+      (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal &&
+        (node.parent.arguments[0].value as string) === "")
```

Comment:
This seems to assume that the first value is the only one that can be empty string.  Later on you are looking at all arguments instead of just the first and anyone of them could be an empty string (or even 'undefined' for that matter).

Reply:
I added these empty case strings in the empty load rule, so we can also flag them and warn the client.

Reply:
I'm not sure how that applies.  You are checking here for no arguments and "", but only for the first argument ('node.parent.arguments[0].value').  Then the next thing you do is look at all the arguments of the function ('node.parent.arguments.forEach(...)'.  I'm suggesting that the check for "" (or even an explicit 'undefined') be done inside the loop.

I realize that it doesn't make sense from what the code should look like, but if that's what were looking at we should be thorough.

Reply:
I see it now . . . it is better to look for the empty property in the rule instead of here since this should just parse the arguments.  If that is the case then you don't need to check for "" here at all.

Reply:
I do agree. Moved all the checks to the empty load case.

Reply:
Are you checking for arguments.length somewhere?  I think you might have a problem calling aruguments.forEach(...) when there are no arguments.

Reply:
It is working even without checking for the length. In the `no-empty-load.test.ts` there are multiple examples with empty load and no problem there.